api/healthCheck/index.ts

```diff
@@ -38,41 +39,41 @@ const healthCheckEndpointDictionary = {
 */
 
 function getPromises(path, req: express.Request): any[] {
-    const isTandCEnabled = showFeature(FEATURE_TERMS_AND_CONDITIONS_ENABLED)
-    HealthCheckUtil.manageTAndCFeature(isTandCEnabled, healthCheckEndpointDictionary)
-    const Promises = []
+    const isTandCEnabled = showFeature(FEATURE_TERMS_AND_CONDITIONS_ENABLED);
+    HealthCheckUtil.manageTAndCFeature(isTandCEnabled, healthCheckEndpointDictionary);
+    const Promises = [];
     if (healthCheckEndpointDictionary[path]) {
         healthCheckEndpointDictionary[path].forEach(endpoint => {
             // TODO: Have health config for this.
-            console.log('healthEndpoints')
-            console.log(healthEndpoints()[endpoint])
-            Promises.push(req.http.get(healthEndpoints()[endpoint]))
-        })
+            console.log('healthEndpoints');
```

Comment:
Can you replace with logger?

Reply:
again we're not looking to increase functionality etc here, just merely protect against a possible P1 issue